bucket/dual-monitor-tools.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "2.7",
+    "description": "Utility for multiple monitor setups.",
+    "homepage": "http://dualmonitortool.sourceforge.net/",
+    "license": "GPL-3.0-only",
+    "url": "https://sourceforge.net/projects/dualmonitortool/files/dualmonitortool/2.7/DualMonitorTools-2.7.zip",
+    "hash": "sha1:3f36099857c08349944dfa3016a65385d8844fc6",
+    "pre_install": [
+        "New-Item -Path \"$dir\\DmtMagicWords.xml\" -ItemType File | Out-Null",
```

Comment:
```suggestion
        "if (-not (Test-Path \"$persist_dir\\DmtMagicWords.xml\")) { New-Item \"$dir\\DmtMagicWords.xml\" -ItemType File | Out-Null }",
```

Reply:
Oh, I forgot this is run on every update.